packages/ember-application/tests/system/engine_instance_initializers_test.js

```diff
@@ -9,7 +9,10 @@ let MyEngine,
 
 function buildEngineInstance(EngineClass) {
   let engineInstance = EngineClass.buildInstance();
-  setEngineParent(engineInstance, {});
+  setEngineParent(engineInstance, {
+    lookup() { return {}; },
+    resolveRegistration() { return {}; }
```

Comment:
I'm undecided whether the parent should be mocked this way, but it seemed preferable to booting a separate full parent app for each test (since the tests are for _engine_ instance initializers).